src/components/Input/index.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from '@emotion/styled';
+import { DefaultProps } from '@utils/types/DefaultProps';
+import { ChangeEventHandler, RefObject } from 'react';
+
+interface InputProps extends DefaultProps<HTMLInputElement> {
+  placeholder?: string;
+  inputRef?: RefObject<HTMLInputElement>;
+  onChange?: ChangeEventHandler<HTMLInputElement>;
```

Comment:
```suggestion
  id: string;
  name?: string;
```

해서 name 없는 경우에는 id 값으로 넣어주게요!